communitiesuk/funding-service-design-authenticator
# BAU: Add flag to disable notification service

### Change description

Adds a condition flag to optionally disable notifications being sent.  Will log the information out instead.  I added test coverage for this module/class as it didn't exist before as well.



### How to test

Check out the branch, set the `DISABLE_NOTIFICATION_SERVICE` flag on development.py to True, request a magic link - the UI should NOT throw an error, and you should see your magic link and other details logged in flask.


### Screenshots

![image](https://user-images.githubusercontent.com/117724519/225613632-f43ef251-79d1-48fd-88c1-d8e61b20783c.png)
